### PR TITLE
[update] About Promscale

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Provided more introductory information about Promscale.

# Links
Fixes https://github.com/timescale/docs/issues/982

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [x] Is the content technically accurate?
- [x] Is the content complete?
- [x] Is the content presented in a logical order?
- [x] Does the content use appropriate names for features and products?
- [x] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

### CODE REVIEW

File: promscale/about-promscale.md [hunk 1]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
 for observability data, processes that data and stores it in TimescaleDB. It
 also provides an interface to query the data with PromQL. The Promscale
 Connector automatically sets up the data structures in TimescaleDB to store the
-data and handles changes in those data structures if required for
-upgrading to newer versions of Promscale.
+data and handles changes in those data structures if required for upgrading to
+newer versions of Promscale. 
+
+Promscale connector is a translator to natively support integrations with OSS
```
Comment (Loquacity) [inline promscale/about-promscale.md:33]:
```suggestion
Promscale connector is a translator that natively support integrations with OSS
```

File: promscale/about-promscale.md [hunk 2]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
+standards such as Prometheus, OpenTelemetry and offers features that are native
```
Comment (Loquacity) [inline promscale/about-promscale.md:34]:
```suggestion
standards such as Prometheus and OpenTelemetry. It includes features that are native
```

File: promscale/about-promscale.md [hunk 3]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
+to Observability ecosystem. Promscale connector creates schema to store metrics
```
Comment (Loquacity) [inline promscale/about-promscale.md:35]:
```suggestion
to an observability ecosystem. Promscale connector creates schemas to store metrics
```

File: promscale/about-promscale.md [hunk 4]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
+and traces. It offers Prometheus endpoint for metrics reads and writes,
```
Comment (Loquacity) [inline promscale/about-promscale.md:36]:
```suggestion
and traces. It offers a Prometheus endpoint for metrics reads and writes,
```

File: promscale/about-promscale.md [hunk 5]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
+OpenTelemetry Line Protocol endpoint to write traces, Jaeger query endpoint to
```
Comment (Loquacity) [inline promscale/about-promscale.md:37]:
```suggestion
an OpenTelemetry Line Protocol endpoint to write traces, and a Jaeger query endpoint to
```

File: promscale/about-promscale.md [hunk 6]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
 **TimescaleDB**: the Postgres-based database where all the observability data is
 stored. It offers a full SQL interface for querying the data as well as advanced
 capabilities like analytical functions, columnar compression and continuous
 aggregates. TimescaleDB offers a lot of  flexibility to also store business and
 other types of data that you can then use to correlate with observability data.
 
+TimecaleDB stores the data and offers the TimescaleDB functionalities to the Promscale
+connector. If you have custom metrics data, the data that is not in the format
+of Prometheus data model you can leverage the Promscale's JSON streaming format
+to store data in Promscale. This offers PromQL for querying metrics from
```
Comment (Loquacity) [inline promscale/about-promscale.md:50]:
```suggestion
connector. If you have custom metrics data, that is not in the Prometheus 
data model format, you can use the Promscale JSON streaming format
to store data in Promscale. This offers PromQL for querying metrics from
```

File: promscale/about-promscale.md [hunk 7]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
+the connector, SQL querying from database.
```
Comment (Loquacity) [inline promscale/about-promscale.md:51]:
```suggestion
the connector, and SQL querying from the database.
```

**Comment by Loquacity** [comment]
Looks like the link checker has found some badly formatted links that will need to be fixed, too.

**Review by Loquacity — APPROVED** [review]

File: promscale/about-promscale.md [hunk 8]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
+and traces. It offers a Prometheus endpoint for metrics reads and writes,
+an OpenTelemetry Line Protocol endpoint to write traces, and a Jaeger query endpoint to
+query traces. Promscale connector manages the complete lifecycle of data stored
+in the database with operations such as compression and retention.
 
 **TimescaleDB**: the Postgres-based database where all the observability data is
 stored. It offers a full SQL interface for querying the data as well as advanced
 capabilities like analytical functions, columnar compression and continuous
 aggregates. TimescaleDB offers a lot of  flexibility to also store business and
 other types of data that you can then use to correlate with observability data.
```
Comment (ramonguiu) [inline promscale/about-promscale.md:45]:
```suggestion
**Promscale Database**:  the database where all the observability data is stored
that combines PostgreSQL with TimescaleDB and the Promscale extension.
It offers a full SQL interface for querying the data, advanced capabilities like
analytical functions, columnar compression and continuous aggregates as well
as specific performance and query experience improvements for observability
data.
```

File: promscale/about-promscale.md [hunk 9]
```diff
@@ -27,15 +27,29 @@ Promscale includes two components:
...
 also provides an interface to query the data with PromQL. The Promscale
 Connector automatically sets up the data structures in TimescaleDB to store the
-data and handles changes in those data structures if required for
-upgrading to newer versions of Promscale.
+data and handles changes in those data structures if required for upgrading to
+newer versions of Promscale. 
+
+Promscale connector is a translator that natively support integrations with OSS
+standards such as Prometheus and OpenTelemetry. It includes features that are native
+to an observability ecosystem. Promscale connector creates schemas to store metrics
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply promscale/about-promscale.md]: Thanks @ramonguiu !!! You can upload the image to s3 and let me know the location, I can update the link in the document.

File: promscale/about-promscale.md [hunk 10]
```diff
@@ -27,14 +27,29 @@ Promscale includes two components:
...
-aggregates. TimescaleDB offers a lot of  flexibility to also store business and
-other types of data that you can then use to correlate with observability data.
+data and handles changes in those data structures if required for upgrading to
+newer versions of Promscale. 
+
+The Promscale Connector is a translator that natively support integrations with OSS
+standards such as Prometheus and OpenTelemetry. It includes features that are native
+to an observability ecosystem. The Promscale Connector creates schemas to store metrics
+and traces. It offers a Prometheus endpoint for metrics reads and writes,
+an OpenTelemetry Protocol endpoint to write traces, and a Jaeger query endpoint to
```
[the remark this answers is not in the document]
Comment (alejandrodnm) [reply promscale/about-promscale.md:37]: Otherwise LGTM

[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply promscale/about-promscale.md:37]: @ramonguiu Can you help in answering the question that @alejandrodnm has?

File: promscale/about-promscale.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (ramonguiu) [reply promscale/about-promscale.md]: I don't have permission to upload to S3. Not sure who takes care of that but usually has been handled by the docs team. The new diagram is available here: https://github.com/timescale/promscale#architecture

[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply promscale/about-promscale.md]: Cool!!! No worries @ramonguiu  I can upload it to S3.

**Review by VineethReddy02 — APPROVED** [review]

**Pull request merged**